g_python/htools.py

```diff
@@ -33,34 +34,90 @@ def __init__(self, ext: Extension, room_users='Users', room_model='RoomReady', r
 
         self.room_users = {}
         self.__callback_new_users = None
+        self.__callback_remove_user = None
 
         self.__ext = ext
         self.__request_id = request
+        self.__lock = threading.Lock()
+        self.lock = self.__lock
```

Comment:
are you accessing this lock from the outside?

Reply:
yes, I don't  know if this is a good idea or not but I needed it because there was some data race happening sometimes